### PR TITLE
Move Makefile to SM2A folder

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
**Summary:** Summary of changes

Moving Makefile to sm2a folder and trigger it from root Makefile

### CODE REVIEW

File: infrastructure/.terraform.lock.hcl
Comment (ciaransweet) [inline infrastructure/.terraform.lock.hcl:1]: Is this meant to be part of this change?

File: sm2a/infrastructure/.terraform.lock.hcl
Comment (ciaransweet) [inline sm2a/infrastructure/.terraform.lock.hcl:1]: Ditto

File: sm2a/Makefile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	docker compose run --rm airflow-cli users create --email airflow@example.com --firstname airflow --lastname airflow --password airflow --username airflow --role Admin
+
+sm2a-local-stop:
+	docker compose down
+
+sm2a-cp-dags:
+	cp -r ../dags .
+
+sm2a-deploy: sm2a-cp-dags
+	@echo "Installing the deployment dependency"
```
Comment (ciaransweet) [inline sm2a/Makefile:52]: `@echo` with `echo` 2 lines below?

Comment (amarouane-ABDELHAK) [reply sm2a/Makefile:52]: Echo for installing reqs and another echo for Deploying SM2A

Comment (ciaransweet) [reply sm2a/Makefile:52]: Probably want `@` then on line 54 for consistency?

Comment (amarouane-ABDELHAK) [reply sm2a/Makefile:50]: Makes sense.

**Review by ciaransweet — APPROVED** [review]

**Pull request merged**